Swiftris/Swiftris/AppDelegate.swift

```diff
@@ -16,31 +16,40 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         // Override point for customization after application launch.
+        NSLog("From AppDelegate: did finish launching")
         return true
     }
 
     func applicationWillResignActive(application: UIApplication) {
         // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
         // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
-    }
+        NSLog("From AppDelegate: did resign active")
+
+        NSNotificationCenter.defaultCenter().postNotificationName("swiftrisDidEnterBackground", object: self)
+}
 
     func applicationDidEnterBackground(application: UIApplication) {
         // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
         // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
+        NSLog("From AppDelegate: did enter background")
     }
 
     func applicationWillEnterForeground(application: UIApplication) {
         // Called as part of the transition from the background to the inactive state; here you can undo many of the changes made on entering the background.
+        NSLog("From AppDelegate: will enter foreground")
     }
 
     func applicationDidBecomeActive(application: UIApplication) {
         // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+        NSLog("From AppDelegate: did become active")
+        
+        NSNotificationCenter.defaultCenter().postNotificationName("swiftrisDidEnterForeground", object: self)
```

Comment:
Same thing as above. Posting a custom notification is not necessary in this scenario.